src/ProgressOnderwijsUtils/Internal/MutableShortStringBuilder.cs

```diff
@@ -0,0 +1,50 @@
+using System.Buffers;
+
+namespace ProgressOnderwijsUtils.Internal;
+
+/// <summary>
+/// faster than StringBuilder since we don't need insert-in-the-middle capability and can reuse this memory
+/// </summary>
+struct MutableShortStringBuilder
```

Comment:
This is unchanged; it's just extracted to a separate file because that makes it clearer this isn't just an sql implementation detail.